moveit_ros/occupancy_map_monitor/CMakeLists.txt

```diff
@@ -15,54 +23,59 @@ if(APPLE)
   find_package(X11 REQUIRED)
 endif(APPLE)
 
-find_package(catkin REQUIRED COMPONENTS
-  moveit_core
-  moveit_msgs
-  pluginlib
-)
-
+find_package(moveit_core REQUIRED)
+find_package(moveit_msgs REQUIRED)
+find_package(pluginlib REQUIRED)
 find_package(Eigen3 REQUIRED)
 find_package(octomap REQUIRED)
-
-catkin_package(
-  INCLUDE_DIRS
-    include
-  LIBRARIES
-    ${MOVEIT_LIB_NAME}
-  CATKIN_DEPENDS
-    moveit_core
-    moveit_msgs
-  DEPENDS
-    EIGEN3
-    OCTOMAP
-)
+find_package(geometric_shapes REQUIRED)
 
 include_directories(include
                     ${Boost_INCLUDE_DIRS}
-                    ${catkin_INCLUDE_DIRS}
+                    ${rclcpp_INCLUDE_DIRS}
```

Comment:
This shouldn't be necessary with ament_target_dependencies

Reply:
Removed `${rclcpp_INCLUDE_DIRS}` from `include_directories`.